media/css/_navigation.scss

```diff
@@ -0,0 +1,127 @@
+.navbar-inner {
+  background: $sidebar-background;
+  border-width: 0 !important;
+}
+
+.navbar-inverse {
+  .navbar-inner {
+    background: $sidebar-background;
+  }
+
+  .brand:hover,
+  .nav > .active > a,
+  .nav > .active > a:hover,
+  .nav > .active > a:focus {
+    background: $sidebar-link-highlight;
+  }
+}
+
+.navbar {
+  .brand {
+    max-height: 50px;
+    overflow: visible;
+    padding: 0 3px;
+    margin-right: 3px;
+    margin-left: 3px;
+  }
+
+  .nav {
+    font-size: 14px;
+    text-align: left;
+    display: inline;
+    margin: 0;
+    list-style: none;
+
+    li {
+      &:hover {
+        background: $sidebar-link-highlight;
+        color: $white;
+      }
+    }
+
+    > li {
+      > ul {
+        padding: 0;
```

Comment:
Properties should be ordered -webkit-box-shadow, box-shadow, display, left, min-width, opacity, padding, position, top, -moz-transition, -ms-transition, -o-transition, transition, -webkit-transiton, visibility